src/preload.js

```diff
@@ -18,37 +27,16 @@ let scripts =
       () => {console.log("add your own scripts!")}
     ]
 
-    const runScripts = () => { scripts.forEach(script => {(script)()}); }
+    const runScripts = () => { scripts.forEach(script => {
+      try {
+        (script)()
+      } catch (_) {}
+    }); }
 
 if (config.allowScripts) runScripts()
 
 document.addEventListener("keydown", (event) => {
-  if (event.code == "Escape") {
+  if (event.key == "Escape") {
     document.exitPointerLock();
   }
-})
-
-window.prompt = () => { // import settings fix
-  var tempHTML = '<div class="setHed">Import Settings</div>';
```

Comment:
Delete all of this why? This is for the prompt for settings, given this is the only instance of prop it’s a workaround. Electron doesn’t like using prompt

Reply:
@AryaveerSR

Reply:
no idk how this works